catalog/wcag2-catalog.yaml

```diff
@@ -0,0 +1,29 @@
+title: WCAG 2.0
+chapters:
+  - id: success_criteria_level_a
+    label: Success Criteria, Level A
+    order: 1
+    criteria:
+      - id: 1.1.1
+        label: Non-text Content (Level A)
+      - id: 1.2.2
+        label: 1.2.2 Captions (Prerecorded) (Level A)
```

Comment:
I think we also need to capture which components this criteria apply to (since some criteria don't apply to some components), so perhaps something like:
```
components: [web, "electronic-docs", software, "authoring-tool"]
```

Reply:
This one would be fine to push to a follow-up PR, if that is simpler.

Reply:
That is a good idea. A follow-up PR would be preferable for me. I will work on that.